fiftyone/operators/delegated.py

```diff
@@ -90,6 +90,8 @@ def set_running(self, doc_id, progress=None, run_link=None):
                 operation
             run_link (None): an optional link to orchestrator-specific
                 information about the operation
+            required_state (ExecutionRunState): an optional required state of the operation. If provided, the
```

Comment:
Typically in our docstrings the default value is in parentheses and the type may be in text in the description.
```python
required_state (None): an optional :class:`ExecutionRunState` as a required current state of the operation.
```